### PR TITLE
R8 to M17

### PR DESCRIPTION
**Pull request by shuds13**
Making documentation policy mandatory.

### CODE REVIEW

File: package_policies/M17.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 **R8.** Each xSDK member package should have sufficient documentation to support use and further development.
 While it is difficult to formalize what constitutes sufficiency of documentation, the requirement will be reviewed by the
-xSDK community based on documentation elements that may include, for example, the following:
+xSDK community based on documentation elements and must include:
   * Instructions on installation and getting started;
   * Documentation on functionality, for example class- and member-level documentation (in object-oriented languages) or more generally API-level documentation;
+
+It is also recommended that the documentation include, for example, the following:
```
Comment (ulrikeyang) [inline package_policies/M17.md]: I suggest to change this to
It is recommended that the documentation also include the following:

Comment (shuds13) [reply package_policies/M17.md]: Agreed

**Review by ulrikeyang — APPROVED** [review]
This is fine.

File: package_policies/M17.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 **R8.** Each xSDK member package should have sufficient documentation to support use and further development.
```
Comment (mgates3) [inline package_policies/M17.md]: This text needs updating as well, R8 => M17, right?

**Review by mgates3 — APPROVED** [review]
Yes, assuming text is renumbered to M17.

**Review by xiaoyeli — APPROVED** [review]

**Review by gardner48 — APPROVED** [review]

**Review by joseeroman — APPROVED** [review]

File: package_policies/M17.md (hunk 2 above)
Comment (shuds13) [reply package_policies/M17.md]: Fixed.

**Review by balay — APPROVED** [review]
Perhaps the corresponding README changes should also be in this PR?

**Review by v-dobrev — APPROVED** [review]
Approving conditionally: should reflect this change in the README.

**Review by masterleinad — APPROVED** [review]

**Review by hartwiganzt — APPROVED** [review]

**Review by stomov — APPROVED** [review]

**Review by jwillenbring — APPROVED** [review]
This is a good thing to require, and to be clear about what is still optional. Approve.

**Review by stomov — APPROVED** [review]

**Review by luszczek — APPROVED** [review]

**Pull request merged**